pkg/test/resourcefixture/testdata/basic/redis/v1alpha1/rediscluster/redisclustersimple/_generated_object_redisclustersimple.golden.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: redis.cnrm.cloud.google.com/v1alpha1
+kind: RedisCluster
+metadata:
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 3
+  labels:
+    cnrm-test: "true"
+  name: rediscluster-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  location: us-central1
+  projectRef:
+    external: ${projectId}
+  pscConfigs:
+  - networkRef:
+      name: network-${uniqueId}
+  replicaCount: 2
+  resourceID: rediscluster-${uniqueId}
+  shardCount: 3
+status:
+  conditions:
+  - lastTransitionTime: "1970-01-01T00:00:00Z"
+    message: The resource is up to date
+    reason: UpToDate
+    status: "True"
+    type: Ready
+  observedGeneration: 3
+  observedState:
+    createTime: "1970-01-01T00:00:00Z"
+    discoveryEndpoints:
+    - address: 10.128.0.2
+      port: 6379
+      pscConfig:
+        networkRef:
+          external: projects/${projectId}/global/networks/network-${uniqueId}
+    preciseSizeGb: 39
+    pscConnections:
+    - address: 10.128.0.2
```

Comment:
nit: This has two `10.128.0.2`.

Reply:
I think we've normalized the address to always be 10.128.0.2, because it's assigned by GCP